ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -217,6 +217,28 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-with-ep-sts-ec2-elb
+- as: aws-ipi-compact-p2-f14
```

Comment:
It's suggested to sort `aws-ipi-compact*` between `aws-ipi-byo*` and `aws-ipi-disconnected*`

Reply:
Thank you for your suggestion. Both profiles have been updated.